addons/google_calendar/utils/google_calendar.py

```diff
@@ -28,6 +28,22 @@ class GoogleCalendarService():
     def __init__(self, google_service):
         self.google_service = google_service
 
+    @requires_auth_token
+    def get_event(self, eventId, sync_token, token=None, timeout=TIMEOUT):
```

Comment:
Any reason it is public ? Otherwise just call it '_get_event' (or '_get_google_event', event better would say).